apps/server/src/modules/tool/uc/context-external-tool.uc.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { ContextExternalToolService, ContextExternalToolValidationService } from '@src/modules/tool/service';
+import { contextExternalToolDOFactory, setupEntities } from '@shared/testing';
+import { ContextExternalToolUc } from '@src/modules/tool/uc/context-external-tool.uc';
+import { Actions, ContextExternalToolDO, EntityId, Permission } from '@shared/domain';
+import { AllowedAuthorizationEntityType, AuthorizationService } from '@src/modules';
+import { ToolContextType } from '@src/modules/tool/interface';
```

Comment:
Please use ../.. when the files are in the module self

Reply:
done